### PR TITLE
Added service blacklist for schedule homepage

### PR DESCRIPTION
**Pull request by votemike**
https://jira.dev.bbc.co.uk/browse/PROGRAMMES-5697

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

File: src/Controller/SchedulesHomeController.php [hunk 1]
```diff
@@ -36,13 +36,19 @@ public function __invoke(ServicesService $servicesService)
 
         $decades = range($earliestDecade, $latestDecade, 10);
 
+        $blacklist = ['p00v5fbq', 'p00qvyk4', 'p00qvyjs'];
```
Comment (BPScott) [inline src/Controller/SchedulesHomeController.php]: The list on PROGRAMMES-5792 had a few more in. I reckon add them all to this list, we can do the smarter "hide the service if it has no broadcasts" thing at a later date, it's unlikely that any of those services will gain broadcasts any time soon.

File: src/Controller/SchedulesHomeController.php [hunk 2]
```diff
@@ -36,13 +36,35 @@ public function __invoke(ServicesService $servicesService)
 
         $decades = range($earliestDecade, $latestDecade, 10);
 
+        // https://jira.dev.bbc.co.uk/browse/PROGRAMMES-5792
+        $blacklist = [
```
Comment (felipeparaujo) [inline src/Controller/SchedulesHomeController.php]: Should this be an attribute?

Comment (votemike) [inline src/Controller/SchedulesHomeController.php]: Like a class property? Is there an advantage to that?

Comment (felipeparaujo) [inline src/Controller/SchedulesHomeController.php]: Honestly don't know. If you think this is fine, then I'm cool with it.

**Comment by BjornTwachtmann** [comment]
looks fine to me

**Pull request merged**